infrastructure/aat.tfvars

```diff
@@ -3,4 +3,5 @@ infrastructure_env = "preprod"
 enable_select_by_case_id = "true"
 createCcdEndpoint = "true"
 jui_base_url = "https://jui-webapp-aat.service.core-compute-aat.internal"
-idam_redirect_url = "https://sscs-case-loader-aat.service.core-compute-aat.internal"
\ No newline at end of file
+idam_redirect_url = "https://sscs-case-loader-aat.service.core-compute-aat.internal"
+appinsights_instrumentation_key = "e2300a5f-a16f-4657-a63b-41913805661e"
```

Comment:
Not sure this is the right way to do this in AKS. Platform Engineering previously advised that app insights key should be loaded in through secrets e.g. https://github.com/hmcts/sscs-ccd-callback-orchestrator/blob/master/charts/sscs-ccd-callback-orchestrator/values.yaml#L19 . You would also need to remove this from Flux as well: https://github.com/hmcts/cnp-flux-config/blob/master/k8s/aat/common/sscs/sscs-cor-backend.yaml#L22